src/inference/include/ie/gpu/gpu_config.hpp

```diff
@@ -94,7 +94,7 @@ DECLARE_GPU_CONFIG_KEY(PLUGIN_THROTTLE);
  * - MEDIUM (DEFAULT) - instructs the GPU Plugin to use any available cores (BIG or LITTLE cores)
  * - HIGH - instructs the GPU Plugin to use BIG cores if they are available
  */
-DECLARE_GPU_CONFIG_KEY(HOST_TASK_PRIORITY);
+DECLARE_CONFIG_KEY(IE_GPU_HOST_TASK_PRIORITY);
```

Comment:
pretty weird name, imho- is it to clearly seperate it from the ??? (maybe worth adding a comment, e.g. recomndationb to use the some 2.0 property

Reply:
Agree, I will align the names between both APIs and handle the difference on plugin side